index.js

```diff
@@ -99,6 +143,28 @@ itemForm.onsubmit = async (event) => {
   document.getElementById('nameInput').focus(); // return focus to the top form input
 };
 
+// might want to rename itemsCOntrol to filter control?
+const toggleItemsControls = () => {
+  // toggle collapsed state
+  isItemsControlHidden = !isItemsControlHidden;
+
+  if (isItemsControlHidden) {
+    // Hide the controls
+    itemsControl.style.display = 'none';
+    toggleMessage.textContent = 'Show Filters';
+    toggleArrow.classList.remove('arrow-down');
+    toggleArrow.classList.add('arrow-up');
+  } else {
+    // Show the controls
+    itemsControl.style.display = 'block';
+    toggleMessage.textContent = 'Hide Filters';
+    toggleArrow.classList.remove('arrow-up');
+    toggleArrow.classList.add('arrow-down');
+  }
+};
+
```

Comment:
I swear I remember learning a couple patterns for this that might be a better approach - I'll need to do some investigation - but this works for MVP